src/main/scala/edu/cmu/cs/obsidian/codegen/CodeGen.scala

```diff
@@ -730,8 +734,21 @@ class CodeGen (val target: Target) {
     }
 
     private def generateMainServerClassMethods(newClass: JDefinedClass, translationContext: TranslationContext): Unit = {
-        newClass._extends(model.directClass("edu.cmu.cs.obsidian.chaincode.ChaincodeBaseMock"))
-        val stubType = model.directClass("edu.cmu.cs.obsidian.chaincode.ChaincodeStubMock")
+
+        System.out.print("Generate mock chaincode: ")
```

Comment:
"mock chaincode: false" is a little surprising when in Hyperledger mode. How about saying what it IS doing instead of what it is NOT doing?